### ISSUE TITLE
Clicking or Releasing Mouse Button Repeats Last Frame of Mouse Movement on Linux

### ISSUE THREAD

**Issue by yoshiyoshyosh** [Yellow-Dog-Man/Resonite-Issues#1349]
### Describe the bug?

On linux screen mode, whenever a mouse button is clicked or released, the last "significant action frame" done by the mouse is repeated. This is usually a mouse movement action, but can also be a scroll action (e.g. scrolling to zoom out an object then releasing may result in another scroll action happening, zooming out the object once more than intended)

this behavior is not observed on wine/proton

### To Reproduce

1. use linux native
2. move the mouse any amount (more noticeable when flicking) or scroll while holding an object
3. observe how clicks or releases repeat the last action done

### Expected behavior

the last action frame does not get duplicated

### Screenshots

https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/36683489/65983089-c977-429f-8b35-1488b3b795ed

### Resonite Version Number

2024.2.12.1430

### What Platforms does this occur on?

Linux

### Log Files

[resonite log](https://gist.github.com/yoshiyoshyosh/d28289f6b9d54ffd4e9bd9025734d53a)

[player.log](https://gist.github.com/yoshiyoshyosh/8b27e62bd69ef27065874f68b405cd1f)

### Reporters

yosh

**Comment by shiftyscales** [comment]
Does this issue occur on the Windows build running through Proton as well, @yoshiyoshyosh?

**Comment by yoshiyoshyosh** [comment]
> Does this issue occur on the Windows build running through Proton as well, [@yoshiyoshyosh](https://github.com/yoshiyoshyosh)?

no. not sure if this is a unity issue or not either, but I guess we'll see again once linux is undeprecated
